src/blas/KokkosBlas_gesv.hpp

```diff
@@ -101,13 +100,25 @@ gesv (const AMatrix& A, const BXMV& B, const IPIVV& IPIV)
       "Valid options include zero-extent 1-D view (no pivoting), or 1-D View with size of "<< A0 << " (partial pivoting).";
     Kokkos::Impl::throw_runtime_exception (os.str ());
   }
-#ifdef KOKKOSKERNELS_ENABLE_TPL_BLAS
+#ifdef KOKKOSKERNELS_ENABLE_TPL_MAGMA //have MAGMA TPL
+  #ifdef KOKKOSKERNELS_ENABLE_TPL_BLAS //and have BLAS TPL
+  if((!std::is_same< typename AMatrix::device_type::memory_space, Kokkos::CudaSpace >::value) &&
```

Comment:
Maybe some comments are needed here. Is my understanding correct that if both MAGMA and BLAS are enabled, any call to `KokkosBlas::gesv` with device views will call the magma gpu interface and any call to `KokkosBlas::gesv` with host views will call the blas interface?

Reply:
@e10harvey You are right. Right now ```KokkosBlas::gesv``` only works with MAGMA TPL and/or BLAS TPL. I added some comments to the file to clarify it.

Reply:
Thanks for your comments, @e10harvey